app/src/main/res/layout/fragment_user_goals_init.xml

```diff
@@ -0,0 +1,122 @@
+<?xml version="1.0" encoding="utf-8"?>
+<FrameLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    tools:context=".userProfileCreation.UserGoalsInitFragment"
+    android:id="@+id/userGoalInitFragment"
+    android:background="@drawable/image_mountain_background"
+    >
+
+    <LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+        xmlns:tools="http://schemas.android.com/tools"
+        android:layout_width="match_parent"
+        android:layout_height="match_parent"
+        android:orientation="vertical"
+        android:layout_gravity="center"
+        tools:context=".userProfileCreation.UserGoalsInitFragment">
+        <TextView
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:layout_gravity="center_horizontal"
+            android:padding="4dp"
+            android:text="@string/to_challenge_you_every_days"
+            android:textIsSelectable="false"
+            android:textSize="28sp"
+            android:textColor="@color/design_default_color_surface"
+            android:textAlignment="center"
+            android:textStyle="bold"/>
+        <TextView
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:layout_gravity="center_horizontal"
+            android:textSize="15sp"
+            android:textStyle="bold"
+            android:padding="5dp"
+            android:text="@string/one_of_the_goal_of_the_drawyourpath_app_is_to_challenge_your_self_every_day_please_enter_some_goals_that_you_will_try_to_follow_as_much_as_possible_each_days"
+            android:textAlignment="center"
+            android:justificationMode="inter_word"
+            android:background="@color/cardview_shadow_start_color"
+            android:textColor="@color/design_default_color_surface"/>
+        <TextView
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:layout_gravity="left"
+            android:layout_marginLeft="5dp"
+            android:textSize="15sp"
+            android:text="@string/how_many_kilometer_do_you_want_to_run_through_each_day"
+            android:textColor="@color/design_default_color_surface"/>
+        <EditText
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:layout_gravity="center_horizontal"
+            android:textSize="20sp"
+            android:id="@+id/input_distanceGoal_text_UserProfileCreation"
+            android:hint="@string/enter_your_distance_goal"
+            android:background="@color/design_default_color_surface" />
```

Comment:
`android:inputType="number"` add this so that only number can be typed

You could also add these so that it stays on 1 line
`android:maxLines="1"`
`android:lines="1"`
`android:singleLine="true"`